clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/tests/GvcfTests.scala

```diff
@@ -117,12 +118,12 @@ trait GvcfTests {
       } yield {
         outputs should be(Seq(expected))
 
-        val storedDocument = getJsonFrom[DocumentGvcf](returnedUpsertId)
-        storedDocument.location should be(expected.location)
-        storedDocument.project should be(expected.project)
-        storedDocument.sampleAlias should be(expected.sampleAlias)
-        storedDocument.version should be(expected.version)
-        storedDocument.gvcfPath should be(expected.gvcfPath)
+        val storedDocument = getJsonFrom(returnedUpsertId)(ElasticsearchIndex.Gvcf)
+        getLocation(storedDocument) should be(expected.location)
+        getStringByName(storedDocument, "project") should be(expected.project)
+        getStringByName(storedDocument, "sampleAlias") should be(expected.sampleAlias)
```

Comment:
These field names will need to be snake case. It might be easier to just strip the bookkeeping fields from `storedDocument`, convert `expected` to JSON, and check their equality.